src/pages/AddPage/index.jsx

```diff
@@ -0,0 +1,216 @@
+
+    setCardNumbers((prev) => {
+      const newState = [...prev];
+      newState[index] = value;
+      return newState;
+    });
+ 
+    if(value.length >= CARD_NUMBER.UNIT_LENGTH) {
+      focusNextNumberInput(index);
+    }
+  };
+
+  const handleDueDate = ({target:{value}}, index) => {
+    if (value.length > DUE_DATE.UNIT_LENGTH) return;
+
+    setDueDate((prev) => {
+      const newState = [...prev];
+      newState[index] = value;
+      return newState;
+    });
+    
+    if(index===0) {
+      setError((prev) => {
+        const newState = [...prev];
+        newState[0] = +value > MONTH.MAX || +value < MONTH.MIN;
+        return newState;
+      });
+    }
+    
+    else {
+      const currentYear = new Date().getFullYear().toString().slice(2);
+      setError((prev) => {
+        const newState = [...prev];
+        newState[1] = +value < currentYear;
+        return newState;
+      });
+    }
+
+    if(value.length >= DUE_DATE.UNIT_LENGTH) {
+      dueYearInputRef.current.focus();
+    }
+  };
+
+  const handleOwner = ({target:{value}}) => {
+    if (value.length > OWNER.MAX_LENGTH) return;
+
+    setOwner(value);
+  };
+
+  const handleCvc = ({target:{value}}) => {
+    if (value.length > CVC.UNIT_LENGTH || isNaN(value)) return;
+
+    setCvc(value);
+  };
+
+  const handlePasswordFirst = ({target:{value}}) => {
+    if (value.length > PASSWORD.UNIT_LENGTH || isNaN(value)) return;
+
+    setPassword((prev) => {
+      const newState = [...prev];
+      newState[0] = value;
+      return newState;
+    });
+
+    if(value.length >= 1) {
+      secondPasswordInputRef.current.focus();
+    }
+  }
+
+  const handlePasswordSecond = ({target:{value}}) => {
+    if (value.length > PASSWORD.UNIT_LENGTH || isNaN(value)) return;
+
+    setPassword((prev) => {
+      const newState = [...prev];
+      newState[1] = value;
+      return newState;
+    });
+  }
+
+  const handleSubmit = () => {
+   if(error[0] || error[1]) {
+      alert('만료일을 확인해주세요');
+      return;
+    }
+    alert(`${owner}님의 카드가 등록되었습니다`)
+  }
+
+  return (
+    <PageWrapper>
+      <Header>
+        <Button><Arrow /></Button>
+        <h2>카드 추가</h2>
+      </Header>
+      <CardWrapper>
+        <Card
+          size="small"
+          company="우테코"
+          cardNumbers={cardNumbers}
+          owner={owner || "NAME"}
+          dueMonth={dueDate.month || 'MM'}
+          dueYear={dueDate.year || 'YY'} 
+        />
+      </CardWrapper>
+      <Form 
+        label='카드 번호' 
+        inputInfo={[
+          {type: 'number', id: 'first-card-number', value: cardNumbers[0], ref: cardNumberInputRefs[0]},
+          {type: 'number', id: 'second-card-number', value: cardNumbers[1], ref: cardNumberInputRefs[1]},
+          {type: 'password', id: 'third-card-number', value: cardNumbers[2], ref: cardNumberInputRefs[2]},
+          {type: 'password', id: 'fourth-card-number', value: cardNumbers[3], ref: cardNumberInputRefs[3]}
+        ]}
+        onChange={handleCardNumber}
```

Comment:
핸들러 이름에도 이벤트가 들어가면 어떨까요?

```suggestion
        onChange={handleCardNumberChange}
```

Reply:
이번 미션에서는 네이밍을 많이 고려하지 못했던 것 같네요. click이나 다른 이벤트를 위한 handler를 추가하기 위해서는 명확하게 바꾸는게 맞겠네요! 제안해주신 방법으로 변경했습니다!! [a063f65](https://github.com/woowacourse/react-payments/pull/90/commits/a063f6599003319fd32a804c5f8dd4c598ed6b11)